install/kubernetes/helm/istio/values.yaml

```diff
@@ -399,3 +401,8 @@ zipkin:
     # requests:
     #  cpu: 100m
     #  memory: 128Mi
+
+# Meshexpansion adds internal load balancers and gateway allowing standalone machines and multiple
+# clusters to connect to the control plane running in the deployed cluster.
+meshexpansion:
+  enabled: true # For testing
```

Comment:
nice!

Reply:
I just noticed this wasn't in a testing file, but the main values.yaml.  Was the intent to force enable mesh expansion for 0.8?